pgctl/cli.py

```diff
@@ -410,7 +410,10 @@ def debug(self):
                 'Must debug exactly one service, not: ' + commafy(self.service_names),
             )
 
-        self.stop()
+        if service.state['state'] != 'down':
+            self.stop()
```

Comment:
This change is unrelated to the goal of the PR -- make pre-start run for debug.

I'd rather we address this bit in a separate PR so this can ship.  If we're deeming the "Service is already stopped" message as useless perhaps we should remove it entirely.